TennisBusiness/View controllers/RegistrationViewController.swift

```diff
@@ -62,20 +88,40 @@ class RegistrationViewController: ScreenViewController, UITextFieldDelegate {
         present(alertController, animated: true, completion: nil)
     }
     
+    private func registerForKeyboardNotifications() {
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillShown), name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillBeHidden), name: UIResponder.keyboardWillHideNotification, object: nil)
+    }
+    
+    @objc private func keyboardWillShown(notification: NSNotification) {
+        guard let userInfo = notification.userInfo else { return }
+        var keyboardFrame:CGRect = (userInfo[UIResponder.keyboardFrameBeginUserInfoKey] as! NSValue).cgRectValue
```

Comment:
Please don't use 'as!'.
Change also `var keyboardFrame:CGRect` to `var keyboardFrame =`